src/kirby/components/modal/services/modal-service-helper.tns.ts

```diff
@@ -0,0 +1,39 @@
+import { ExtendedShowModalOptions } from 'nativescript-windowed-modal/windowed-modal.common';
+import { Injectable } from '@angular/core';
+import { ModalDialogService } from 'nativescript-angular/directives/dialogs';
+import { ViewContainerRef } from '@angular/core';
+
+import { ModalComponent } from '~/kirby/components/modal/modal.component';
+import { ModalConfig } from '~/kirby/components/modal/config/modal-config';
+import { ModalCloserService } from './modal-closer-service';
+
+@Injectable()
+export class ModalServiceHelper {
```

Comment:
The naming of this file is not conformant with the default naming convention, eg.:
`modal-service-helper.tns.ts` => `modal.service-helper.tns.ts` or `modal-service.helper.tns.ts`